Jenkinsfile

```diff
@@ -545,10 +553,10 @@ pipeline {
                     }
                     agent {
                         dockerfile {
-                            filename 'Dockerfile.centos.7'
-                            dir 'utils/docker'
+                            filename 'utils/docker/Dockerfile.centos.7'
                             label 'docker_runner'
-                            additionalBuildArgs dockerBuildArgs() +
+                            additionalBuildArgs dockerBuildArgs(add_repo: false,
+                                                                cachebust: false) +
```

Comment:
This means you will never update the image being used for this stage, using ever more outdated tools as time goes by, until somebody/thing (which is nothing automated at the moment) forces an image refresh in some way.

Our stance (until current at least) is that we always want the image we are working on to be up-to-date _to the minute_, which we do by updating it (in a layer near the end, to leverage caching) on every use, through the use of `CACHEBUST`.

Are you proposing we change this stance?

Reply:
You're (almost) correct and that isn't quite what I'd intended.  I had intended to be able to use docker caches efficiently *outside* of CI, but using them inside seems like a good idea so I included it.

Our use of CACHEBUST was inconsistent, in that it didn't apply to ubuntu, and actually we don't build CentOS 8 at all in CI, and in master it only applies to yum packages, not pip packages which are installed long before the CACHEBUST statement.

This PR does have change changes I didn't intent, but it sounds like master was also not behaving intended either, I'll have a think about this overnight.